AnkiDroid/src/test/java/com/ichi2/anki/OnboardingFlagTest.kt

```diff
@@ -0,0 +1,98 @@
+/****************************************************************************************
+ *                                                                                      *
+ * Copyright (c) 2021 Shridhar Goel <shridhar.goel@gmail.com>                           *
+ *                                                                                      *
+ * This program is free software; you can redistribute it and/or modify it under        *
+ * the terms of the GNU General Public License as published by the Free Software        *
+ * Foundation; either version 3 of the License, or (at your option) any later           *
+ * version.                                                                             *
+ *                                                                                      *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY      *
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A      *
+ * PARTICULAR PURPOSE. See the GNU General Public License for more details.             *
+ *                                                                                      *
+ * You should have received a copy of the GNU General Public License along with         *
+ * this program.  If not, see <http://www.gnu.org/licenses/>.                           *
+ ****************************************************************************************/
+
+package com.ichi2.anki
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.ichi2.anki.OnboardingUtils.isAlreadyVisited
+import com.ichi2.anki.OnboardingUtils.setAsVisited
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class OnboardingFlagTest : RobolectricTest() {
+
+    lateinit var context: Context
+
+    @Before
+    fun init() {
+        context = ApplicationProvider.getApplicationContext()
```

Comment:
defined by RobolectricTest

```suggestion
        context = getTargetContext() 
```

Reply:
Updated.